### ISSUE TITLE
NVDA2024.1 beta 7. Windows nt services is inaccessible

### ISSUE THREAD

**Issue by amirmahdifard** [nvaccess/nvda#16188]
before, i reported this problem when i was testing a pr build of nvda, but someone told me that pr builds are not sighned, but i see the same problem in nvda 2024.1 beta7. When going to windows NT services list, everything is totally inaccessible, and we can't do anything there.
### Steps to reproduce:
1: press windows key, and search services.
2: klick on the item that says services, app.
### Actual behavior:
everything inside there is inaccessible
### Expected behavior:
everything should be accessible, like 2023.3
#### NVDA installed/portable/running from source:
portable
#### NVDA version:
2024.1 beta7
#### Windows version:
Windows 10 22H2 (AMD64) build 19045.4046 enterprise
#### Name and version of other software in use when reproducing the issue:
services nt service
#### Other information about your system:
i don't know what other info from my system do you need here
#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
2023.3.3 and older works perfectly fine
#### If NVDA add-ons are disabled, is your problem still occurring?
yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes

**Comment by amirmahdifard** [comment]
also disk management has the same problem, i don't know how many more viaraces dialogs has this bug

**Comment by amirmahdifard** [comment]
also computer management

**Comment by amirmahdifard** [comment]
it seemse any administrative app becomes inaccessible. I ran my teamtalk as administrator and it became inaccessible

**Comment by amirmahdifard** [comment]
i ran my portable nvda.exe file as administrator and the problem semze resolved. i'll let you know if any update result i get

**Comment by XLTechie** [comment]
Portables have many of the same problems as unsigned builds. Try it installed.

**Comment by amirmahdifard** [comment]
@CyrilleB79 hi, i'm sorry to ask this unrelated thing here, i wanted to open a new  issue to ask my question there, but i didn't. I wanted to ask, in a pull request of nvda, when the ap viare completes, how can i get the build of that pr? last time i asked for a build, and you gave me the link for it. So, how can i get the build link for any pull request in NVAcess? thanks a lot!

**Comment by amirmahdifard** [comment]
i looked at all AppVeyor things but i couldn't find the build link

**Comment by seanbudd** [comment]
This is a limitation of portable builds
